File: src/main/java/life/yolbi/util/IMinecraftClient.java
package life.yolbi.util;

import net.minecraft.client.MinecraftClient;

/**
 * @author yuxiangll
 * @since 2024/7/5 下午8:21
 * IntelliJ IDEA
 */
public interface IMinecraftClient {
    MinecraftClient mc = MinecraftClient.getInstance();

//    default boolean fullNullCheck() {
//        return mc.player == null || mc.world == null ;
//    }

}

File: src/main/kotlin/today/yapeteam/utils/IMinecraftClient.kt
package today.yapeteam.utils

import net.minecraft.client.MinecraftClient

/**
 * @author yuxiangll
 * @since 2024/6/7 下午8:42
 * IntelliJ IDEA
 */
interface IMinecraftClient {
    val mc: MinecraftClient
        get() = MinecraftClient.getInstance()
}